app/models/concerns/admin_statuses.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
```

Comment:
I hate to drop this on you after you wrote all this, but I did find some code we could potentially use instead:
https://github.com/psu-libraries/etda_workflow/blob/main/app/presenters/admin/submission_form_view.rb#L19-L35
The wording for these are a little different, but the admins should still recognize these.  Also, the `@session` param/variable in there can be removed.  Using the `SubmissionFormView` might seem a little out of place in the custom reports, but at least we wouldn't be adding more code.

I could go either way.  The `AdminStatuses` seems a bit cleaner to me, however it is extra code that could be done by the `SubmissionFormView`.